src/background.js

```diff
@@ -57,7 +58,8 @@ var background = (function(){
                                     target_url: target_url
                                 },
                                 message:      "[" + branch_name + "] " + "@" + project_event.data.user_name + " " + display_id + " " + commit_message + " (" + project_event.data.total_commits_count + " commits)",
-                                current_time:  project_event.created_at || new Date()
+                                current_time: project_event.created_at || new Date(),
+                                author_email: project_event.data.user_email
```

Comment:
Use `project_event.author_id` instead of `project_event.data.user_email` 

Because `data.user_email` is contained in only commit event, but `author_id` is contained in always